site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -148,6 +152,22 @@ is determined based on the priority of each rule and the operations specified.
   `append` rules from the same extension.
 - If a rule has removed a header, then lower priority rules cannot further modify the header.
 
+### Indexing
+
+Static rulesets are indexed by the browser before they can be used. For large numbers of static
```

Comment:
This is also true for session/dynamic rulesets. So just Rulesets.

Reply:
I've removed this sentence, so done... kinda!